test/parallel/test-child-process-validate-stdio.js

```diff
@@ -5,14 +5,13 @@ const common = require('../common');
 const assert = require('assert');
 const _validateStdio = require('internal/child_process')._validateStdio;
 
-const expectedError =
-  common.expectsError({ code: 'ERR_INVALID_OPT_VALUE', type: TypeError }, 3);
+const expectedError = { code: 'ERR_INVALID_OPT_VALUE', type: TypeError };
```

Comment:
@BridgeAR I think it's done!

That was an awesome review! I learned a lot! Thank you